docker-app/qfieldcloud/core/tests/utils.py

```diff
@@ -148,3 +154,95 @@ def wait_for_project_ok_status(project: Project, wait_s: int = 30):
 
 def fail(msg):
     raise AssertionError(msg or "Test case failed")
+
+
+class QfcFilesTestCase(APITransactionTestCase):
```

Comment:
I suggest we change this to a mixin and leave the actual test user to choose what *TestCase class they want to use. Can be done in a follow-up PR too.